.github/workflows/tests.yml

```diff
@@ -36,18 +36,30 @@ jobs:
         run: |
           npx zx scripts/org-setup.mjs
 
-      - name: Run browser tests
+      - name: Run browser tests with 3 attempts
+        uses: nick-fields/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd
```

Comment:
use this wrapper https://github.com/salesforcecli/github-workflows/commit/2e9e588c0d8cc1fbd6324438fa9f70e3ec35532f

it's the same, but means consistency across all repos and only updating the SHA in one place

Reply:
oh, wait, it's missing the param.